tests/TestCase/View/Widget/CheckboxWidgetTest.php

```diff
@@ -219,7 +219,7 @@ public static function uncheckedProvider(): array
      * @param mixed $checked
      */
     #[DataProvider('uncheckedProvider')]
-    public function testRenderUnCheckedValue($checked): void
+    public function testRenderUnCheckedValue(string|int|bool|null $checked): void
```

Comment:
Not sure if for tests this is worth doing
If so the documented types are now out of sync